azure/scope/cluster.go

```diff
@@ -25,7 +25,8 @@ import (
 	"strconv"
 	"strings"
 
-	asonetworkv1 "github.com/Azure/azure-service-operator/v2/api/network/v1api20220701"
+	asonetworkv1api20201101 "github.com/Azure/azure-service-operator/v2/api/network/v1api20201101"
```

Comment:
There are multiple 2020 versions for the network API group that we'll end up using eventually, so I'd vote to keep these as-is just so we don't have to get too creative to avoid collisions. 

https://azure.github.io/azure-service-operator/reference/network/